app.py

```diff
@@ -1,25 +1,32 @@
-from flask import Flask, request, render_template
-import requests
+
+from flask import Flask, request, render_template , url_for , redirect  
+import requests  
+
+from markupsafe import escape
 
 app = Flask(__name__, template_folder='template')
 
 @app.route('/')
 def index():
-    return render_template("index.html")    
+    return render_template("index.html")
 
 
-@app.route('/', methods=['POST'])
+@app.route('/', methods=['POST'])       
 def Onsubmit():
-    place = request.form['place']
+    place = request.form['place']                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   
     API_KEY = 'e03b0e4011564a1a9e11eb7086d70d1e'
```

Comment:
Usually these type secrets we should not directly use in the code.

Reply:
okay